cmd/booster-bitswap/run.go

```diff
@@ -125,3 +138,18 @@ func getBoostAPI(ctx context.Context, ai string) (api.Boost, jsonrpc.ClientClose
 
 	return api, closer, nil
 }
+
+func dataDirPath(ctx *cli.Context) string {
+	dataDir := ctx.String("data-dir")
```

Comment:
The "data-dir" flag doesn't seem to be defined in the run command.
I'd suggest using cmd.FlagRepo for consistency with the `boost` client.